assets/respec.css

```diff
@@ -234,6 +234,10 @@ aside.example .marker > a.self-link {
   color: inherit;
 }
 
+.example pre {
```

Comment:
We have an examples.css. Might make sense to move it there.m

Reply:
ah! good point.